app/build.gradle

```diff
@@ -9,22 +9,48 @@ android {
         targetSdkVersion 25
         versionCode 1
         versionName "1.0"
-        testInstrumentationRunner "android.support.test.runner.AndroidJUnitRunner"
+        multiDexEnabled true
+
+        testApplicationId "com.nervousfish.nervousfish.test"
+        testInstrumentationRunner "com.nervousfish.nervousfish.test.Instrumentation"
     }
+
+    sourceSets {
+        androidTest {
+            assets.srcDirs = ['src/androidTest/assets']
+        }
+    }
+
     buildTypes {
+        debug {
+            testCoverageEnabled true
+        }
+
         release {
             minifyEnabled false
             proguardFiles getDefaultProguardFile('proguard-android.txt'), 'proguard-rules.pro'
         }
     }
+
+    packagingOptions {
+        exclude 'LICENSE.txt'
+    }
+
+    lintOptions {
+        abortOnError false
```

Comment:
Why did you add this, you didn't do anything related to linting (right)? Still, I would say we want to abort on a lint error (at least on Travis)